components/form/textInput/src/index.js

```diff
@@ -0,0 +1,59 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const renderErrorMessage = error => (
+  error && (
+    <div className='sui-FormTextInput-errorMessage'>
+      <span className='sui-FormTextInput-errorMessageLabel'>{error}</span>
+    </div>
+  )
+)
+
+const FormTextInput = ({className, errorMessage, name, onChange, placeholder, value}) => (
```

Comment:
You are adding this class in your other PR too, please remove wherever it has to be removed

Reply:
Yes, this is the right branch for these files, I have just removed from the other PR, thanks!